### DISCUSSION TITLE
I have some problem with my server

### DISCUSSION THREAD

**Discussion by MRcreeper176** [efroemling/ballistica#665] [Q&A]
My server was working correctly after i run it, but after 2 days...
I tried to change my port (it was 1176), but it doesn't work.
Where's the problem from?
![20240224_192723](https://github.com/efroemling/ballistica/assets/156354493/b4ba227c-6d99-4225-ad35-73b79f3d8b6b)

**Comment by Loup-Garou911XD** [comment]
You need to forward that port from your router

**Reply by MRcreeper176** [reply]
How to do it?
Sorry if i'm asking too much😅

**Reply by Loup-Garou911XD** [reply]
Depends on your router

**Reply by Loup-Garou911XD** [reply]
Google something like `port forwarding on your_router_name`

**Comment by MRcreeper176** [comment]
![Uploading 17098357517672932478234635855994.jpg…]()


Did you mean this?

**Reply by MRcreeper176** [reply]
If it was, i did it when i was setting up my server

**Comment by MRcreeper176** [comment]
Server is working😐
😂😂😅

**Reply by MRcreeper176** [reply]
But it's not public yet

**Reply by Loup-Garou911XD** [reply]
Yeah that wasn't a error, it was just a warning telling you it's not accessible from internet because port is not open

**Reply by MRcreeper176** [reply]
How can i make it public?
It's true in the config file, but it isn't public yet.

**Reply by Loup-Garou911XD** [reply]
Forward port 1177 in your router, also I would recommend to use some bigger number because there's a possibility that port could be used by some other service

**Reply by MRcreeper176** [reply]
Which port number do you recommend?

**Reply by MRcreeper176** [reply]
For the best performance?

**Reply by Loup-Garou911XD** [reply]
It doesn't affect performance, but I recommend the default 43210 port

**Reply by MRcreeper176** [reply]
I did it, but when i wanted to connect to my server(manually) it says (host is in another party)

**Reply by Loup-Garou911XD** [reply]
Hmm try restarting it

**Reply by MRcreeper176** [reply]
It's running again, but it is not public yet🤐
